data/remnant ships.txt

```diff
@@ -348,6 +347,87 @@ ship "Gull" "Gull (Bunks)"
 
 
 
+ship "Ibis"
+	sprite "ship/ibis"
+	thumbnail "thumbnail/ibis"
+	attributes
+		category "Medium Warship"
+		licenses
+			"Remnant Capital"
+		"cost" 30000000
+		"shields" 22000
+		"hull" 11000
+		"required crew" 20
+		"bunks" 48
+		"mass" 300
```

Comment:
This seems rather light, especially when you compare it to the heavier Pelican that is worse in HP by quite a bit (33k vs 17k). I'd say reduce the health of this carrier to be closer to that of the Pelican.

Reply:
Reduced both Shields and hull a bit, and upped the mass by 50 so its hull mass is now 30 higher than the Pelican, rather than 20 smaller.